pkg/config/config.go

```diff
@@ -50,10 +50,11 @@ type Example struct {
 }
 
 type Config struct {
-	Build   *Build `json:"build" yaml:"build"`
-	Image   string `json:"image,omitempty" yaml:"image"`
-	Predict string `json:"predict,omitempty" yaml:"predict"`
-	Train   string `json:"train,omitempty" yaml:"train"`
+	Build       *Build `json:"build" yaml:"build"`
+	Image       string `json:"image,omitempty" yaml:"image"`
+	Predict     string `json:"predict,omitempty" yaml:"predict"`
+	Train       string `json:"train,omitempty" yaml:"train"`
+	Concurrency int    `json:"concurrency,omitempty" yaml:"concurrency"`
```

Comment:
Part of me wonders if this should be an object with a single key (`max`) so that it can be extended in future if needed.

Reply:
that sounds like a pretty reasonable idea, what other keys do you think it might have?